src/components/Tooltip/Content.tsx

```diff
@@ -23,6 +23,11 @@ export const Content = ({
   avoidCollisions = true,
   ...props
 }: TooltipContentProps) => {
+  // Stop showing empty tooltip content when children is null, undefined or empty string
+  if (!children) {
+    return null;
```

Comment:
Should be checked within the type system and require children always otherwise force the user to not render tooltip at all?

Reply:
CC: @krzysztofzuraw